recipes/libctl/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "libctl" %}
+{% set version = "4.0.0" %}
+{% set sha256 = "b0f46ed29acd13be542a4988d7f0604b10869d6b4c41916c539dc99711fb5458" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/stevengj/{{ name }}/releases/download/v{{version}}/{{ name }}-{{version}}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - gcc
```

Comment:
I think if you have a build dep on gcc you need to add a run dep on libgcc, since you are creating shared libraries

Reply:
Fixed.